.github/styles/Google/Acronyms.yml

```diff
@@ -0,0 +1,70 @@
+extends: conditional
+message: "Spell out '%s', if it's unfamiliar to the audience."
+link: 'https://developers.google.com/style/abbreviations'
+level: warning
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '\b([A-Z]{3,5})\b'
+second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
+# ... with the exception of these:
+exceptions:
+  - API
+  - ASP
+  - AWS
+  - CLI
+  - CPU
+  - CSS
+  - CSV
+  - CUDA
+  - DEBUG
+  - DOM
+  - DPI
+  - ETL
+  - FAQ
+  - GCC
```

Comment:
This may be a stupid comment, but how about GCP and GCE?

Reply:
Ah, good catch. Looks like all our uses of "GCP" and "GCE" are in special blocks like dropdowns and tabs. Those widgets aren't linted due to a limitation of Vale.

Fixed.